### DISCUSSION TITLE
Add support for Suno ai for audio generation

### DISCUSSION THREAD

**Discussion by shhlife** [griptape-ai/griptape#1525] [Ideas]
- [x ] I have read and agree to the [contributing guidelines](https://github.com/griptape-ai/griptape#contributing).
Can we have an audio generation driver for SunoAI for audio generation?

https://aimlapi.com/models/suno-ai

cheers,
Jason

**Comment by collindutter** [comment]
This is an interesting use-case, we should rename `TextToSpeechDriver`s to `AudioGenerationDriver`s.

**Comment by shhlife** [comment]
Curious why this was moved to a discussion? Is there a reason why we wouldn't want to create a driver to handle audio generation?
